macOS/DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/NetworkProtectionNotificationsPresenterFactory.swift

```diff
@@ -19,22 +19,25 @@
 import AppLauncher
 import Foundation
 import NetworkProtection
+#if !NETP_SYSTEM_EXTENSION
+import VPNNotifications
```

Comment:
Needed for the VPN app extension (not system extension).  The appex is built but not longer used - it's behind a feature flag we might remove in the future.